matchers/matchers.go

```diff
@@ -0,0 +1,73 @@
+package matchers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/quay/zlog"
+	"go.opentelemetry.io/otel/baggage"
+	"go.opentelemetry.io/otel/label"
+
+	"github.com/quay/claircore/libvuln/driver"
+	_ "github.com/quay/claircore/matchers/defaults"
+	"github.com/quay/claircore/matchers/registry"
+)
+
+type Configs map[string]driver.MatcherConfigUnmarshaler
+
+type Matchers struct {
+	// provides run-time matcher construction.
+	factories map[string]driver.MatcherFactory
+	// configs provided to matchers once constructed.
+	configs Configs
+	client  *http.Client
+	// out-of-tree matchers
+	matchers []driver.Matcher
+}
+
+type MatchersOption func(m *Matchers)
+
+// NewManager will return a manager ready to have its Start or Run methods called.
```

Comment:
stale comments. 

Also out of curiosity why did you use the Manager pattern used for a long running update manager here?

Reply:
I've implemented it based on `updates/manager.go`, but it doesn't exhibit the behaviour of manager, it is still a factory of factory.